ddl/constant.go

```diff
@@ -77,6 +78,7 @@ const (
 		start_time bigint,
 		state_update_time bigint,
 		meta longblob,
+		error BLOB,
 		summary json,
-		unique key(namespace, task_key))`
```

Comment:
Why delete origin uk?

Reply:
1. namespace not used, so the subtask (nil, task_key) will be duplicated
2. the uk has been deleted from `BackgroundSubtaskTableSQL`, you can see L49-65